test/common/cdp.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
```

Comment:
Not necessary, I took care to only call `print` with a single argument. I. e. this will work with both Python 2 and 3. (Harmless though, if you want to keep it)

Reply:
Ah neat trick! Dropped.